SzczepionkaAPI/src/main/java/com/szczepionka/entity/Patient.java

```diff
@@ -24,6 +22,9 @@
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
 
+    @Column(updatable = false, nullable = false, unique = true)
```

Comment:
Is there any reason why have you used `@Column` with attribute `nullable = false`?
Comparing to the `@NotNull` annotation, which is used in our other enitites, its behaviour is slightly different and to be honest I don't see any reason why should we use this attribute over the annotation.
It is better to rely on the Bean Validation standard

Reply:
I'm aware of the difference. The DB should be always the exact representation of the code, when done in @Collumn annotation the constraint are added to the tables in DB. But I agree that @NotNull annotation also should be used as it will never allow incorrect value to be set. Not sure wihich approach is better, or maybe should we use them both?